vuepress/docs/next/docs/curate/bundle-details.md

```diff
@@ -184,6 +185,16 @@ See the [Plugin Environment Variables](../../tutorials/devops/plugin-environment
     }
    ]
 ```
+#### Configure a Path for Static Assets
+To configure your micro frontend with access to static assets, Entando provides two paths, one for widgets and another for EPCs, defined in a custom element.  
```

Comment:
remove "defined in a custom element". This is about micro frontends and isn't connected to HTML custom elements per se.